robot/Cumulus/tests/browser/manage_households/add_newcontact_managehh.robot

```diff
@@ -1,31 +1,44 @@
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/ManageHouseHoldPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
 Suite Teardown  Delete Records and Close Browser
 
+*** Keywords ***
+
+Setup Test Data
+    &{contact} =         API Create Contact                    Email=${EMAIL1}
+    Store Session Record  Account                               &{contact}[AccountId]
+    Set suite variable    &{contact}
+    ${first_name} =       Generate Random String
+    Set suite variable    ${first_name}
+    ${last_name} =        Generate Random String
+    Set suite variable    ${last_name}
+
+*** Variables ***
+${EMAIL1}       user1@automation.com
+
 *** Test Cases ***
 
 Add New Contact to Existing Household 
-    [tags]  unstable
-    &{contact} =  API Create Contact    Email=skristem@robot.com
-    Store Session Record    Account    &{contact}[AccountId]
-    Go To Record Home  &{contact}[AccountId] 
-    # Click Link    link=Show more actions
-    Click Link    link=Manage Household    
-    Sleep     5     Input-textbox-notloaded-properly    
-    Choose Frame   Manage Household
-    ${first_name} =           Generate Random String
-    ${last_name} =            Generate Random String
-    Search Field By Value    Find a Contact or add a new Contact to the Household    ${first_name} ${last_name}
-    Wait For Locator    manage_hh_page.button    New Contact
-    Click Button    text=New Contact
-    Wait For Locator    span_button    New Contact    
-    Click Span Button    New Contact
-    Click Button       title=Save
-    Unselect Frame
-    Wait Until Page Contains    Account Owner
-    Wait For Record To Update    &{contact}[AccountId]    &{contact}[LastName] and ${last_name} Household
-    Select Tab    Related
-    Load Related List    Contacts
-    Verify Related List Items    Contacts    ${first_name} ${last_name}
+
+    Go To Page                          Details
+    ...                                 Account
+    ...                                 object_id=&{contact}[AccountId]
+
+    Click Link                          link=Manage Household
+    Go To Page                          Custom                            ManageHousehold
+
+    Add contact                         New                               ${first_name} ${last_name}
+    Current Page Should Be              Details                           Account
+    Wait Until Page Contains            Account Owner
+    Wait For Record To Update           &{contact}[AccountId]             &{contact}[LastName] and ${last_name} Household
+    Select Tab                          Related
+    Load Related List                   Contacts
+    Verify Related List Items           Contacts                          ${first_name} ${last_name}
```

Comment:
you could move this load related list keyword inside verify related list items keyword

Reply:
Done